src/LintProcess.ts

```diff
@@ -3,12 +3,17 @@ import { spawn, ChildProcess } from 'child_process';
 import * as path from 'path';
 
 export default class LintProcess {
-    public Process: ChildProcess;
+    private Process: ChildProcess;
 
-    public StdOut: string = '';
-    public StdErr: string = '';
+    private StdOut: string = '';
+    //private StdErr: string = '';
```

Comment:
Just delete the code instead of commenting it out if its not needed anymore, it'll be in the git history anyways